lm75bd/lm75bd.c

```diff
@@ -26,7 +26,39 @@ error_code_t lm75bdInit(lm75bd_config_t *config) {
 }
 
 error_code_t readTempLM75BD(uint8_t devAddr, float *temp) {
-  /* Implement this driver function */
+  error_code_t errCode;
+  
+  if (temp == NULL) {
+    return ERR_CODE_INVALID_ARG; 
+  }
+  
+  uint8_t temp1 = TEMP_ADDR;
+  
+  errCode = i2cSendTo(devAddr, &temp1, 1);
+  if (errCode != ERR_CODE_SUCCESS)
+  {
+    return errCode; 
+  }
```

Comment:
Replace with `RETURN_IF_ERROR_CODE(i2cSendTo(devAddr, &temp1, 1));`. This will log and return the error code if `i2cSendTo` fails.